### DISCUSSION TITLE
Mid-Summer Pages Check-in

### DISCUSSION THREAD

**Discussion by LiteBrite82** [community/community#134988] [Other Feature Feedback, Questions, & Ideas]
![This banner image shows a moonscape scene with two Ocotcats standing on the moon posing with the earth in the background. It includes the text "Pages Check-in" in the upper left corner.](https://github.com/community/community/assets/92824518/25aa9c5d-f8ae-4b2f-98eb-b895e5343d19)
> Alt Text: This banner image shows a moonscape scene with two Ocotcats standing on the moon posing with the earth in the background. It includes the text "Pages Check-in" in the upper left corner.

# Hello Pages community! Let’s do a mid-summer check-in

## 🧑‍💻 August Community Topic
**What is your favorite front-end tech stack**

- Looking for a way to connect with the community? [Write a quick “blog style” discussion](https://github.com/orgs/community/discussions/new?category=pages ) about your favorite Front End Tech Stack and how you use it. Show off your work and methods, share your workflow, get feedback, and grow together!

  - **Suggested content:** 
    - Share what you’ve built   
    - Why you are using your selected stack  
    - Likes and dislikes of the stack
    - What was your deployment experience (with Pages or other)  
    - What you learned  
    - Tips for other community members  
    - Ask for feedback / help on your project 

### Thank you to all of our great contributors! These were the most active contributors in Pages last month.

- **@akruse10** and **@knarrff** took on some 404 errors involving DNS settings, DNS verification / DNS takeover, and Create-React-App configuration for Gh-Pages in a 3 week battle! [View the hard-fought victory deployment](https://samkruse.com/).
  -  [Custom apex domain 404 error #132013](https://github.com/orgs/community/discussions/132013)
  <br/>

- **@maggieWillDesign** and **@CleverLemming1337** took on a tricky 404 error where it seemed that new files pushed to the repo did not show up in the deployment. It turns out they needed a `.nojekyll` file to keep jekyll from renaming files! [Check out the Deployment](https://maggiewilldesign.github.io/) 
  - [404 error #132723](https://github.com/orgs/community/discussions/132723)
<br/>

- **@ChristianC561** made an important discovery around the `NotServedByPagesError` and SquareSpace. [SquareSpace now defaults to the `www` variant of DNS naming schemas](https://support.squarespace.com/hc/en-us/articles/206541837-Removing-the-www-from-your-domain). So even though a user may expect their domain name to be `example.com`, with SquareSpace it is actually [`www.example.com`](http://www.example.com); this is super important when selecting the `Custom Domain Name` setting in `Pages` settings. 
  - [I for some reason cannot connect my github pages and squarespace custom domain. NotServedByPagesError #133994](https://github.com/orgs/community/discussions/133994)
<br/>

- **@SuhainaFathimaM** Faced down the `Dangerous Site` flag from Google, with help from **@davevad93**, **@Damandeep-S**, **@knarrff**, and **@aishamc**. Look out for naming conventions in your repo names; using names like `netflix_clone` can get your domain flagged (\<username\>.github.io) by Google. This results in all deployments associated with that domain being labeled as dangerous.
  - [My all deployment pages in github are marked as dangerous sites.. but they are just simple projects like calculator and my portfolio etc.. #133058](https://github.com/orgs/community/discussions/133058)

### **[Sunsetting GitHub Pages’ legacy worker](https://github.blog/changelog/2024-07-08-pages-legacy-worker-sunset/)**

- #### .nojekyll

  - GitHub Pages uses jekyll (behind the scenes) to deploy every project. Jekyll likes to rename some files to comply with its file structure. This can cause problems for projects not built with jekyll.   
  - TL;DR: If you are not using jekyll in your project, and you are not using the Actions Build Method, add an empty file named `.nojekyll` to your `main` branch (or whichever branch you run your build from).   
    - If you are not using a Front End Framework (just plain .html, .css, .md, and .js files) still use a `.nojekyll` file in your `main` branch to avoid any issues.   
    - If you are getting odd 404s in your deployed application (and are not using jekyll), add a `.nojekyll` file for good measure. 
      

- #### GitHub Actions Deployment

    
  - Github Actions is the new default method of deployment for Pages (Deploy from a branch is still available for now)  
      
    - Using Actions for deployment brings your app into the CI/CD world  
      - Any push to `main` will automatically update your deployment. Or you can customize your workflow any way you want!   
      - [You will need to have actions enabled in your account](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository\#managing-github-actions-permissions-for-your-repository), and adjust your actions permissions to a level appropriate to your needs.

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by Lall-091** [comment]
****
